Made the instructions clearer (#46)
TykTechnologies/tyk-helm-chart: README.md

The Tyk owned MDCB registry is private and requires adding users to our organisation which you then define as a secret when pulling the MDCB image. Please contact your account manager to arrange this.

## Install Tyk Hybrid Gateways (This can be used either for Multi-Cloud Gateways or MDCB slaves)
To install, first modify `values_hybrid.yaml` file as follows:

> 1. Add redis passwordi in redis.pass. It's the value of $REDIS_PASSWORD (The host should be "tyk-redis-master.tyk-ingress.svc.cluster.local" if you used the tyk-ingress as the namespace.

Reviewer: ```suggestion
1. Add redis password in `redis.pass` value. It's the value of `$REDIS_PASSWORD` environment variable (the host should be "tyk-redis-master.tyk-ingress.svc.cluster.local" if you used the tyk-ingress as the namespace.
```

> 2. Add your RPC key (i.e. ORGID) in tyk_k8s.org_id

Reviewer: ```suggestion
2. Add your RPC key (i.e. ORGID) in `tyk_k8s.org_id` value
```

Author: @excieve  - I removed **(i.e. ORGID)**

> 3. Add your API KEY in tyk_k8s.dash_key (Could be the api key of any dashboard user but better to have a dedicated one) 

Reviewer: ```suggestion
3. Add your API key in `tyk_k8s.dash_key` value (could be the API key of any dashboard user but better to have a dedicated one) 
```

> 4. Add your dashboard url in tyk_k8s.dash_url. If it's a multi cloud account the value *https://admin.cloud.tyk.io* is already set for you.

Reviewer: ```suggestion
4. Add your dashboard URL in `tyk_k8s.dash_url` value. If it's a Tyk SaaS account the value *https://admin.cloud.tyk.io* is already set for you.
```

Author: Why `Tyk SaaS` and not `Tyk Multi Cloud`?

Reviewer: We don't have Multi Cloud anymore. All SaaS accounts are hybrid.
